ui/src/app.js

```diff
@@ -1783,14 +1789,14 @@ function setCurrentDiscount (currentDiscount) {
   if (currentDiscount > 0) {
     $('#insert-first-bill-promo-button').hide()
     $('#choose-fiat-promo-button').hide()
-    $('#insert-first-bill-coupon-added').html(`✔ Coupon added (${currentDiscount}% discount)`)
-    $('#choose-fiat-coupon-added').html(`✔ Coupon added (${currentDiscount}% discount)`)
-    $('#insert-first-bill-coupon-added').show()
-    $('#choose-fiat-coupon-added').show()
+    $('#insert-first-bill-code-added').html(`✔ Promo code added (${currentDiscount}% discount)`)
+    $('#choose-fiat-code-added').html(`✔ Promo code added (${currentDiscount}% discount)`)
```

Comment:
Rafael said this will break multi language support, "will not generate translation"

Reply:
It is correct, going to fix that on the PR related to translation